web/blog/vast-v2.4/index.md

```diff
@@ -1,54 +1,131 @@
 
-- basic docker compose env that we plan to extend in the future
-- outline what we plan to do with this; more in future blog posts
+As of this release, VAST no longer supports *writing* to its old proprietary
+storage format, but will still support *reading* from it until the next major
+release. In the background, VAST transparently rebuilds old partitions to take
+advantage of the new format without any downtime. This may cause some additional
+load when starting VAST first up after the update, but ensures that queries run
+as fast as possible once all old partitions have been converted.
 
-## Web Plugin
+If you want to know more about Feather and Parquet, check out our in-depth blog
+post series on them:
 
-- basic rest endpoint + web ui that we plan to extend in the future
-- outline what we plan to do with this; more in future blog posts
+1. [Enabling Open Investigations][parquet-and-feather-1]
+2. [Writing Security Telemetry][parquet-and-feather-2]
+3. Coming soon
 
-## Python Bindings
+[parquet-and-feather-1]: /blog/parquet-and-feather-enabling-open-investigations/
+[parquet-and-feather-2]: /blog/parquet-and-feather-writing-security-telemetry/
 
-- state of the art: most of the security ecosystem interfaces to python
-- new python bindings with good arrow integration make it easy to work with the
-  data
-- show small pandas example for rendering an arrow table
+## What's Next?
 
-## Other Changes
+VAST v2.4 contains a few new and experimental toys to play with. Here's an
+overview of what they are, and how they all make it easier to integrate VAST
+with other security tools.
 
-- build scaffolding produces debian packages
-- rebuild now also rebatches
-- load all plugins by default now
-- new metrics keys: ingest-total, catalog.num-events-total,
-  catalog.num-partitions-total, index.memory-usage, catalog.memory-usage,
-  filesystem.*
-- metrics uds hang fix
+### Docker Compose
+
+A new set of [Docker Compose files][docker-compose] makes it easier than ever to
+get started with VAST. This is not designed for high-performance deployments of
+VAST, but rather to make it easier to try VAST out—all-batteries included,
+because we want to use this to showcase and test the myriad of integrations
+in a modern SOC.
+
+Our vision for this is to show how VAST as a modular platform can power modern
+and sustainable approaches to composable security.
+
+[docker-compose]: /docs/setup/deploy/docker-compose
+
+### REST API and Frontend User Interface
+
+The experimental `web` plugin adds a [REST API][rest-api] to VAST, and also a
+frontend user interface we [built in Svelte][frontend-code].
+
+Both the API and the frontend are still considered unstable and subject to
+change without notice. We plan to stabilize and version the API in the future.
+Fundamentally, the API serves two purposes:
+
+1. Make it easier to write integrations with VAST
+2. Serve as a backend for VAST's bundled frontend
+
+TODO: Add screenshot of UI
```

Comment:
@dit7ya do you happen to have a nice looking one?

Reply:
Added one.

Reply:
Nice, I would just expand the plugins for the curious viewer.